ntbs-service/wwwroot/css/notificationView.scss

```diff
@@ -58,6 +58,28 @@
   padding-right: 50px;
 }
 
+.overview-treatment-events-table {
+  & th, td {
+    font-size: 16px;
+  }
```

Comment:
We've got some logic in _ManualTestResultTable for this purpose.. see `nhsuk-table--reduced-font`.